### DISCUSSION TITLE
Activate wifi on Banana Pro board with Debian

### DISCUSSION THREAD

**Discussion by Tlisbel** [johang/sd-card-images#117] [Q&A]
Hello, I have a Banana Pro board that runs Debian Stable through an SD card.  I am trying to activate the wifi on the board, according to the Lemaker user manual, the ap6210 driver must be installed with the command "sudo modprobe ap6210" but it answers "modprobe: FATAL: Module ap6210 not found in directory /lib/modules/6.1.0 -10-armmp".  In addition, Wlan0 is not shown when running Iwconfig.  What I can do?

**Comment by johang** [comment]
You probably have to install firmware for the WLAN chip.

https://wiki.debian.org/brcm80211
